server/cluster/unsafe_recovery_controller_test.go

```diff
@@ -314,7 +315,7 @@ func TestFailed(t *testing.T) {
 		recoveryController.HandleStoreHeartbeat(req, resp)
 		applyRecoveryPlan(re, storeID, reports, resp)
 	}
-	re.Equal(failed, recoveryController.GetStage())
+	re.Equal(finished, recoveryController.GetStage())
```

Comment:
why it would be finished

Reply:
`An error is triggered` -> `change stage to exit force leader` -> `u.err is cleaned up` -> `exit force leader succeed` -> `finished since u.err is nil`

Reply:
Oh, that's definitely unexpected. I misunderstood your meaning before.